lambda_functions/book_room.py

```diff
@@ -0,0 +1,77 @@
+"""
+book_room
+
+Function performing the read/write operations of the API Resource /booking [POST]
+"""
+
+import boto3
+from datetime import datetime
+import time
+
+
+DATE_FORMAT = "%Y-%m-%d"
+dynamodb = boto3.resource('dynamodb')
+
+def to_time(epoch):
+    import datetime 
+    date_time = datetime.datetime.fromtimestamp( epoch ) 
+    return date_time
+
+def lambda_handler(event, context):
+    #TODO implement
+    response=None
+    r_table = dynamodb.Table('rooms')
+    r_dynamodata=r_table.scan()['Items']
+    valid_rooms=[]
+    for i in r_dynamodata:
+        if i['room_type']==event['room_type']:
+            valid_rooms.append(i)
+    print(valid_rooms)
+    
+    a = datetime.strptime(event['date_from'], DATE_FORMAT)
+    b = datetime.strptime(event['date_to'], DATE_FORMAT)
+    delta = b - a
+    difference_dates = delta.days
+    print(difference_dates)
+    
+    a_table = dynamodb.Table('availability')
+    a_dynamodata=a_table.scan()['Items']
+    a_list=[]
+    start = event['date_from']
+    end = event['date_to']
+    while(start!=end):
+        for day in a_dynamodata:
+            if start==day['date']:
+                a_list.append(day)
+        start = to_time(datetime.strptime(start, DATE_FORMAT).timestamp()+86400)
+        start = start.strftime(DATE_FORMAT)
+        print(start)
+    print(a_list)
```

Comment:
remove unnecessary print statements unless the actual result needs to b displayed.